react-components/src/hooks/useClickedNode.tsx

```diff
@@ -43,16 +44,21 @@ export const useClickedNodeData = (): ClickedNodeData | undefined => {
       cadIntersection?.treeIndex
     ).data ?? [];
 
-  if (cadIntersection === undefined || nodeData.length === 0) {
-    return undefined;
-  }
+  useEffect(() => {
+    if (cadIntersection === undefined || nodeData.length === 0) {
```

Comment:
I feel like we should remove the second condition and set `clickedNodeData` with undefined `fdmNode`, `view` and probably `cadNode` when we click on non-mapped object. And the reason for that is that with this hook we are basically creating a "react-y" API for `viewer.on('click', ...)` and if we won't add capabilities for normal non-mapped clicks the user will be forced to have both in place if they want to just access, let's say, a clicked cursor position on the model or treeIndex for styling. I think it would be really useful to have it. Since with current behaviour when you have `clickedNodeData` as a dependency in useEffect hook when you click twice on non-mapped part of the model the hook is updated only for the first time if previously some mapped node was selected.